### PR TITLE
Added important resources for Data Science.

### PR DESCRIPTION
**Pull request by AKSingh-Udacity**
Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

**Comment by AKSingh-Udacity** [comment]
This is my commit to open source project under Hacktoberfest!

File: src/pages/data-science-tools/index.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ In this section, we'll have guides to a wide variety of tools used by data scien
 Data scientists are inquisitive and often seek out new tools that help them find answers. They also need to be proficient in using the tools of the trade, even though there are dozens upon dozens of them. Overall, data scientists should have a working knowledge of statistical programming languages for constructing data processing systems, databases, and visualization tools. Many in the field also deem a knowledge of programming an integral part of data science; however, not all data scientist students study programming, so it is helpful to be aware of tools that circumvent programming and include a user-friendly graphical interface so that data scientists’ knowledge of algorithms is enough to help them build predictive models.
 
 R remains the leading tool, with 49% share, but Python grows faster and almost catches up to R. RapidMiner remains the most popular general Data Science platform. Big Data tools used by almost 40%, and Deep Learning usage doubles.
+Data Science is OSEMN O= Obtain , S= Scrub,M=Model, N= iNterpret the Data.
```
Comment (ezioda004) [inline src/pages/data-science-tools/index.md]: Hi, can you add proper indentation? Like: O = Obtain, S = Scrub, M = Model etc.

File: src/pages/data-science-tools/index.md [hunk 2]
```diff
@@ -8,3 +8,5 @@ In this section, we'll have guides to a wide variety of tools used by data scien
...
+There is one good resource for Data Science and Machine Learning by Open Source Data Science Masters.Follow on github  datasciencemasters!!!*[Resouses for Data Science](https://github.com/datasciencemasters/go)
```
Comment (ezioda004) [inline src/pages/data-science-tools/index.md]: 1 Indentation before "Follow" as well.
Also, you have a typo in "Resouses". Move that to a new line aswell. Thanks.

**Comment by paulywill** [comment]
@AKSingh-Udacity Good work, and congrats on your first-time contribution!
CONGRATS

**Pull request merged**